ttnn/cpp/ttnn/operations/conv2d/device/kernels/reader_conv_activations_2d_mcast_padded_with_halo_3x3_weights_v2.cpp

```diff
@@ -46,6 +46,7 @@ void kernel_main() {
     constexpr uint32_t window_inner = get_compile_time_arg_val(7);
     constexpr uint32_t act_block_h_datums = get_compile_time_arg_val(8);
     constexpr uint32_t weight_size_w = get_compile_time_arg_val(10);
+    constexpr uint32_t act_block_w_extra_align_bytes = get_compile_time_arg_val(12);
```

Comment:
is this to align the sticks to 16B?

Reply:
Yes, It is to align boundry to 32 bytes of single line(input_channels X 9) so that next pixel starts in next row of (n*h*w).